app/assets/stylesheets/pathways/_show_page.scss

```diff
@@ -0,0 +1,120 @@
+.Container {
+   display: flex;
+   flex-direction: row;
+}
+
+.Title {
+   font-size: 50px;
+   font-weight: bold;
+   letter-spacing: 2px;
+   margin-bottom: 10px;
+}
+
+.Difficulty {
+   text-align: center;
+   padding: 3px;
+   width: 4cm;
+   font-weight: lighter;
+   letter-spacing: 1px;
+   border-radius: 10px;
+   font-size: smaller;
+   margin-bottom: 15px;
+}
+
+.Difficulty.beginner {
+   background-color: rgb(202, 240, 188);
+}
+
+.Difficulty.intermediate {
+   background-color: rgb(252, 209, 156);
+}
+
+.Difficulty.advanced {
+   background-color: rgb(255, 172, 167);
+}
+
+.title4 {
+   font-size: 30px;
+   font-weight: bold;
+}
+
+.c {
+   margin-top: 4%;
+   margin-right: 7%;
+   padding-left: 6%;
+}
+
+.ChallengesHeader {
+   display: flex;
+   flex-direction: row;
+   align-items: center;
+   margin-left: 45%;
+   background-color: rgb(233, 238, 255);
+   height: 110px;
+   width: 25cm;
+   border-top-left-radius: 13px;
+   border-top-right-radius: 13px;
+}
+
+.add {
+   color: white;
+   letter-spacing: 2px;
+   background-color: rgb(51, 85, 209);
+   height: 50px;
+   width: 160px;
+   border-radius: 20px;
+   text-align: center;
+   padding-top: 14px;
+   font-size: smaller;
+   margin-left: 9cm;
+}
+
+.create {
+   color: white;
+   letter-spacing: 2px;
+   background-color: rgb(51, 85, 209);
+   height: 50px;
+   width: 200px;
+   border-radius: 20px;
+   text-align: center;
+   padding-top: 14px;
+   font-size: smaller;
+   margin-left: 15px;
+}
+
```

Comment:
.create and .add are identical apart from the margin left how can we keep this DRY?